modules/op-constructing-pipelines-using-pipeline-builder.adoc

```diff
@@ -0,0 +1,43 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc
+
+[id="op-constructing-pipelines-using-pipeline-builder_{context}"]
+= Constructing Pipelines using the Pipeline Builder
+
+You can use the Pipeline Builder to:
+
+* Construct a Pipeline flow using existing Tasks and CLusterTasks. When you install the OpenShift Pipelines Operator, it adds reusable Pipeline ClusterTasks to your cluster.
+* Specify the type of resources required for the Pipeline execution, and if required, add additional parameters to the Pipeline.
+* Reference these Pipeline resources in each of the Tasks in the Pipeline as input and output resources.
+* The parameters for a Task are prepopulated based on the specifications of the Task. If required, reference the additional parameters added to the Pipeline in the Task.
+
+.Procedure
+
+. In the *Add* view of the *Developer* perspective, click the *Pipeline* tile to see the *Pipeline Builder* page.
+. Enter a unique name for the Pipeline.
+. Select a Task from the *Select task* list to add a Task to the Pipeline. For this example, select a *s2i-nodejs* Task.
```

Comment:
I would suggest rewriting this sentence: 
"For this example, select a *s2i-nodejs* Task."

To something like:
"This example uses a *s2i-nodejs* Task." or "This example is based on selecting a *s2i-nodejs* Task."